moduleframework/common.py

```diff
@@ -42,24 +42,82 @@
 from avocado.utils import process
 from moduleframework.mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
 
+
+def print_info(*args):
+    """
+    Print information from the expected stdout and
+    stderr files from the native test scope.
+
+    See `Test log, stdout and stderr in native Avocado modules
+    <https://avocado-framework.readthedocs.io/en/latest/WritingTests.html
+    #test-log-stdout-and-stderr-in-native-avocado-modules>`_ for more information.
+
+    :param args: object
+    :return: None
+    """
+    for arg in args:
+        print(arg, file=sys.stderr)
+
+
+def print_debug(*args):
+    """
+    Print information from the expected stdout and
+    stderr files from the native test scope if
+    the **DEBUG** envvar is set to True.
+
+    See `Test log, stdout and stderr in native Avocado modules
+    <https://avocado-framework.readthedocs.io/en/latest/WritingTests.html
+    #test-log-stdout-and-stderr-in-native-avocado-modules>`_ for more information.
+
+    :param args: object
+    :return: None
+    """
+    if is_debug():
+        print_info(*args)
+
+
+
+class MTFConfParser(object):
+    configname = "mtf.yaml"
+    source_cfg_file = os.path.join(os.path.dirname(__file__), configname)
+    default_cfg_file = os.path.join("/etc", configname)
+    user_cfg_file = os.path.expanduser("~/." + configname)
+    options = None
+
+    def __init__(self):
+        config = {}
+        for cfgfile in [self.source_cfg_file, self.default_cfg_file, self.user_cfg_file]:
+            if os.path.exists(cfgfile):
+                print_info("MTF config loading: {}".format(cfgfile))
+                config.update(yaml.load(open(cfgfile)))
+        assert config.get("generic")
+        self.options = config
+
+
+    def get(self, section=None, key=None):
```

Comment:
What about to use function `getattr` like https://docs.python.org/2/library/functions.html#getattr

Reply:
I'll use that. sounds well